app/portainer/hostmanagement/open-amt/open-amt.service.ts

```diff
@@ -1,66 +1,88 @@
 import axios, { parseAxiosError } from 'Portainer/services/axios';
 
 import {
-  AMTConfiguration,
+  OpenAMTConfiguration,
   AMTInformation,
   AuthorizationResponse,
   Device,
-  DeviceFeatures
+  DeviceFeatures,
 } from './model';
 
 const BASE_URL = '/open_amt';
 
-export async function configureAMT(formValues : AMTConfiguration) {
+export async function configureAMT(formValues: OpenAMTConfiguration) {
   try {
     await axios.post(`${BASE_URL}/configure`, formValues);
   } catch (e) {
     throw parseAxiosError(e as Error, 'Unable to configure AMT');
   }
 }
 
-export async function getAMTInfo(endpointId : number) {
+export async function getAMTInfo(endpointId: number) {
```

Comment:
call it environmentId and use https://github.com/portainer/portainer/blob/07e7fbd270f8d5c3f83da8e17a7704d6bec31e13/app/portainer/environments/types.ts#L1 for its type. (will make it easier to change the type for ids in the future)

Reply:
same for the rest of the file